packages/cli/templates/vue2/src/App.vue

```diff
@@ -3,12 +3,12 @@
 </template>
 
 <script>
-import FieldPlugin from './components/FieldPlugin.vue'
+import _FieldPlugin from './components/FieldPlugin.vue'
 import FieldPluginExample from './components/FieldPluginExample/index.vue'
 
 export default {
   components: {
-    FieldPlugin,
+    // FieldPlugin,
```

Comment:
How does this affect the template? If we are not using the FieldPlugin as component isn't it better to remove it than comment it?

Reply:
Actually that `FieldPlugin` component is just a bare minimum empty component where users can start writing their code. And `FieldPluginExample` is a replacement to demo the functionality. In our README, we guide them to remove `FieldPluginExample` and start writing code into `FieldPlugin` component. So we sort of keep it there, but not entirely remove.